guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,190 @@
+[id="Upgrading-Project-from-EL7-to-EL8{context}"]
+ifdef::satellite[]
+= Upgrading {Project} from {RHEL} 7 to {RHEL} 8
+endif::[]
+ifdef::katello,foreman-el[]
+= Upgrading {Project} from Enterprise Linux 7 to Enterprise Linux 8
+endif::[]
+
+Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
+LEAPP is the supported tool for {RHEL} 7 to {RHEL} 8 upgrades.
+
+.Prerequisites
+ifdef::katello,foreman-el[]
+* {Project} {ProjectVersionPrevious} running on CentOS Linux 7.
+- {Project} installations running on {RHEL} 7 cannot upgrade to {RHEL} 8. You must have a {RHEL} compatible LEAPP build to upgrade.
+endif::[]
+
+* Access to available repositories or a local mirror of repositories.
+* No outstanding updates.
+ifdef::katello,foreman-el[]
+* {Project} installations can be upgraded to CentOS Stream 8 and a {RHEL} rebuild.
+endif::[]
+
+.Procedure
+ifdef::katello,foreman-el[]
+. Configure the https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/[@theforeman/leapp COPR Repository], which contains newer LEAPP packages than those shipped by https://wiki.almalinux.org/elevate/[AlmaLinux/ELevate], and support {Project} upgrades:
+
++
+----
+# curl -o /etc/yum.repos.d/theforeman-leapp.repo https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/repo/epel-7/group_theforeman-leapp-epel-7.repo
+----
+. Install required packages:
++
+----
+# yum install leapp leapp-repository leapp-data-centos
+----
+
+ifdef::katello,foreman-el[]
+. Add {Project} specific repositories to `/etc/leapp/files/leapp_upgrade_repositories.repo`:
++
+----
+[leapp-foreman]
+name=Foreman 3.2
+baseurl=https://yum.theforeman.org/releases/3.2/el8/$basearch
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet7-release
```

Comment:
This sounds off to me. Is is meant to be the same GPG as below?

Reply:
@bangelic Can you verify this with a SME?

Reply:
I can verify that this is completely wrong and should be `gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman`

Reply:
@evgeni That's the verification needed haha. Thank you!